app/models/manageiq/providers/kubevirt/infra_manager/vm.rb

```diff
@@ -17,8 +17,17 @@
 class ManageIQ::Providers::Kubevirt::InfraManager::Vm < ManageIQ::Providers::InfraManager::Vm
   include_concern 'Operations'
 
+  POWER_STATES = {
+    'Running'    => 'on',
+    'Pending'    => 'powering_up',
+    'Scheduling' => 'powering_up',
```

Comment:
@rmohr Please check whether I added here all the statuses that are possible for VMs. I based my list on pod life cycle https://blog.openshift.com/kubernetes-pods-life/

Reply:
Right now we have these: https://github.com/kubevirt/kubevirt/blob/master/pkg/api/v1/types.go#L303. The only one you are missing is `Scheduled`. That one does not exist on pods.

I also saw now that these fields are not reflected in the api-reference: http://www.kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachinestatus. Will file an issue in kubevirt.

Reply:
And we should probably rethink our `Scheduled` and `Pending` and `Scheduling` meaning. We slightly diverge here from Pods, with out necessarily for a good reason. But for now it works like that.

Reply:
For reference: https://github.com/kubevirt/kubevirt/issues/1085

Reply:
Great, I will update it with `Scheduled`. Feel free to open an issue if anything changes with vm statuses.